ros_buildfarm/templates/snippet/publisher_xunit.xml.em

```diff
@@ -23,19 +24,15 @@ assert type_tag in ('CTestType', 'GoogleTestType', 'JUnitType'), 'Unsupported te
       <thresholds>
         <org.jenkinsci.plugins.xunit.threshold.FailedThreshold>
           <unstableThreshold>0</unstableThreshold>
-          <unstableNewThreshold/>
-          <failureThreshold/>
-          <failureNewThreshold/>
         </org.jenkinsci.plugins.xunit.threshold.FailedThreshold>
-        <org.jenkinsci.plugins.xunit.threshold.SkippedThreshold>
-          <unstableThreshold/>
-          <unstableNewThreshold/>
-          <failureThreshold/>
-          <failureNewThreshold/>
-        </org.jenkinsci.plugins.xunit.threshold.SkippedThreshold>
+        <org.jenkinsci.plugins.xunit.threshold.SkippedThreshold/>
       </thresholds>
       <thresholdMode>1</thresholdMode>
       <extraConfiguration>
         <testTimeMargin>3000</testTimeMargin>
+        <sleepTime>0</sleepTime>
+        <reduceLog>false</reduceLog>
+        <followSymlink>true</followSymlink>
```

Comment:
There are also
several new extra configuration fields for a sleep timer between reports (to avoid
report CPU time starving out the Jenkins ping thread) a stdout/stderr
log reduction option to save memory during report generation, and an
option whether or not to follow symlinks when reviewing reports. I have
opted to leave all of these at their defaults for now.